app/models/forem/subscription.rb

```diff
@@ -9,7 +9,12 @@ def send_notification(post_id)
       # If a user cannot be found, then no-op
       # This will happen if the user record has been deleted.
       if subscriber.present?
-        SubscriptionMailer.topic_reply(post_id, subscriber.id).deliver
+        mail = SubscriptionMailer.topic_reply(post_id, subscriber.id)
```

Comment:
I'm considering dropping support for earlier versions of Rails so that we won't have this method at all. It will always be `deliver_later`, and then `ActiveJob` can handle it.